packages/aila/src/utils/language/findAmericanisms.ts

```diff
@@ -1,8 +1,9 @@
+// eslint-disable-next-line @typescript-eslint/triple-slash-reference
```

Comment:
I couldn't get Typescript to find this file, and it's a bit of a distraction so using a /// here for now